hazelcast/src/main/java/com/hazelcast/durableexecutor/impl/operations/RetrieveAndDisposeResultOperation.java

```diff
@@ -21,9 +21,12 @@
 import com.hazelcast.durableexecutor.impl.DurableExecutorDataSerializerHook;
 import com.hazelcast.nio.Bits;
 import com.hazelcast.spi.BlockingOperation;
+import com.hazelcast.spi.ReadonlyOperation;
 import com.hazelcast.spi.WaitNotifyKey;
+import com.hazelcast.spi.impl.MutatingOperation;
 
-public class RetrieveAndDisposeResultOperation extends DisposeResultOperation implements BlockingOperation {
+public class RetrieveAndDisposeResultOperation extends DisposeResultOperation implements BlockingOperation,
+        ReadonlyOperation, MutatingOperation {
```

Comment:
It's kind of from two worlds, it's a read & write operation in one (since it disposes the result, so a subsequent read won't get it). It would be a bummer if you could invoke it under READ quorum not met, since to me you read the value also. 
I guess it's a matter of policy...

Reply:
Fixed.